src/storybook/tokenDefaults.ts

```diff
@@ -0,0 +1,52 @@
+import { Network, TokenDetails } from 'types'
+
+export const networkIds = Object.values(Network).filter(Number.isInteger) as Network[]
+
+export const defaultNetworkId = Network.Mainnet
+
+// All Default Tokens
+export const tokenList: TokenDetails[] = CONFIG.initialTokenList.map(
+  ({ id, name, symbol, addressByNetwork, decimals = 18 }) => ({
+    id,
+    name,
+    symbol,
+    address: addressByNetwork[defaultNetworkId] || '0x',
+    decimals: decimals,
+  }),
+)
+
+// Default params to be used as initial values
+// and when there's no Token found for symbol
+export const defaultBaseToken: TokenDetails = {
```

Comment:
This two mocked constants are used for many stories, I've made this PR for it:
https://github.com/gnosis/dex-react/pull/1386